selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestOrganizationServiceClient.java

```diff
@@ -46,14 +47,10 @@ public TestOrganizationServiceClient(
   }
 
   public List<OrganizationDto> getAllRoot() throws Exception {
-    return getAll(null);
-  }
-
-  public List<OrganizationDto> getAll(@Nullable String parent) throws Exception {
     List<OrganizationDto> organizations =
         requestFactory.fromUrl(getApiUrl()).request().asList(OrganizationDto.class);
 
-    organizations.removeIf(o -> !o.getParent().equals(parent));
+    organizations.removeIf((o -> nonNull(o.getParent())));
```

Comment:
Look like there are odd parentheses in the command: 
`organizations.removeIf((o -> nonNull(o.getParent())));`
\>
`organizations.removeIf(o -> nonNull(o.getParent()));`